docs/docs/training-data-format.mdx

```diff
@@ -685,3 +685,72 @@ rasa test
 
 If you want to know more about testing head over to
 [Testing Your Assistant](testing-your-assistant.mdx).
+
+
+## End-to-end Training
+
+:::caution experimental feature
+End-to-end training is an experimental feature.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+With [end-to-end training](stories.mdx#end-to-end-training), you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline.
+Instead, you can simply put the text of the user message directly in the stories,
+by using `user:` key.
```

Comment:
For the sake of consistency, we might want to use "key" only for the actual word, not for the following colon.

```suggestion
by using `user` key.
```